src/utils/hash.ts

```diff
@@ -53,6 +61,21 @@ export function getSelectorFromName(funcName: string) {
   return toHex(starknetKeccak(funcName));
 }
 
+/**
+ * Function to get hex selector from function name, decimal string or hex string
+ * @param value hex string | decimal string | string
+ * @returns Hex selector
+ */
+export function getSelector(value: string) {
+  if (isHex(value)) {
```

Comment:
can these if-s be replaced by calling the whole `getHexString` function from number.ts?

Reply:
I would need to use try catch to use it as I need extended input type 'string', which would be a super weird implementation.